pulse2percept/implants/base.py

```diff
@@ -520,26 +524,51 @@ def _make_grid(self, spacing, x, y, z, rot, names, etype, **kwargs):
             y_arr = (np.arange(rows) * spc - (rows / 2.0 - 0.5) * spc)
             x_arr, y_arr = np.meshgrid(x_arr, y_arr, sparse=False)
         elif self.type.lower() == 'hex':
-            # Hexagonal grid from x,y coordinates:
-            x_arr_lshift = (np.arange(cols) * spc - (cols / 2.0 - 0.5) * spc -
-                            spc * 0.25)
-            x_arr_rshift = (np.arange(cols) * spc - (cols / 2.0 - 0.5) * spc +
-                            spc * 0.25)
-            y_arr = (np.arange(rows) * np.sqrt(3) * spc / 2.0 -
-                     (rows / 2.0 - 0.5) * spc)
-            x_arr_lshift, y_arr_lshift = np.meshgrid(x_arr_lshift, y_arr,
-                                                     sparse=False)
-            x_arr_rshift, y_arr_rshift = np.meshgrid(x_arr_rshift, y_arr,
-                                                     sparse=False)
-            # Shift every other row to get an interleaved pattern:
-            x_arr = []
-            for row in range(rows):
-                if row % 2:
-                    x_arr.append(x_arr_rshift[row])
-                else:
-                    x_arr.append(x_arr_lshift[row])
-            x_arr = np.array(x_arr)
-            y_arr = y_arr_rshift
+            # When orientation is horizontal, x_arr is shifting to create the hex grid
+            if orientation == 'horizontal':            
```

Comment:
I like the split between 'horizontal' and 'vertical', but there seems to be something wrong with the grid:

![image](https://user-images.githubusercontent.com/5214334/82471752-fd53bd80-9a7b-11ea-8884-6a4f32a38285.png)

You can see that the distance between columns is too large. Not sure if that is a problem with both orientations. Please track down the error, and when you find it, also add a test that makes sure this issue is fixed.